razorpay-quick-payments.php

```diff
@@ -298,6 +299,8 @@ function wpCheckRazorpayResponse()
                     $this->message = 'Thank you for shopping with us. However, the payment failed.\n' . $error ;
                 }
 
+                $this->message = "<script>var displayRzpModal = '" . $this->message . "';</script>";
```

Comment:
@abdulwahidsharief please add comment for the reason why this message is added as script tag.